arangod/RestHandler/RestReplicationHandler.h

```diff
@@ -61,7 +61,9 @@ class RestReplicationHandler : public RestVocbaseBaseHandler {
     if (len >= 1) {
       std::string const& command = suffixes[0];
       if (command == AddFollower || command == HoldReadLockCollection ||
-          command == RemoveFollower || command == LoggerFollow) {
+          command == RemoveFollower || command == LoggerFollow ||
+          command == Batch || command == Inventory || command == Revisions ||
+          command == Dump) {
         return RequestLane::SERVER_REPLICATION_CATCHUP;
```

Comment:
when command is "make follower", does it not have to catchup? Is it different behavior from adding a follower?

Reply:
To be honest, I do not know what this `make-follower` API is used for. It is definitely not used in the cluster to catch up a follower. Therefore it is OK to leave it out here with the special prioritities. In any case, I could not find any usage of this API or any documentation of it. It might be useful for some active/failover deployment mode or something like this, or even single server asynchronous replication. I do not know.